events/moose/moose.go

```diff
@@ -610,8 +622,11 @@ func (s *Subscriber) NotifyDisconnect(data events.DataDisconnect) error {
 			connectionTime, // seconds
 			"",
 			int32(-1),
-		))
+		)); err != nil {
+			return err
+		}
 	}
+	return s.response(moose.NordvpnappSetContextApplicationNordvpnappConfigCurrentStateIsOnVpnValue(false))
```

Comment:
why is this needed if for connect was not?

Reply:
I simply followed the logic that this has to be `true` only after regular VPN connection but after disconnect this value always has to be `false`. Yes, it would likely be correct if this was in the regular disconnect branch as it is never set to `true` from meshnet connections but I kept it here to cover more area for recover in case some misconfiguration happened.